src/components/POLine/FundDistribution/FundDistributionView.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
```

Comment:
try to import only needed modules, like `import toString from 'lodash/toString';`